package.json

```diff
@@ -31,11 +34,15 @@
     "babel-preset-env": "1.6.1",
     "babel-preset-react": "6.24.1",
     "css-loader": "1.0.1",
+    "gulp": "3.9.1",
+    "gulp-eslint": "^5.0.0",
+    "gulp-sass-lint": "^1.4.0",
```

Comment:
All dependencies should have fixed versions in application projects. You don't want to have unexpected break changes if a dependencies gets updated to a newer version when you reinstall the modules.

Reply:
You are right. This has happened to me in a previous project. Thanks for letting me know.

Reply:
👍